pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -144,6 +144,7 @@ typedef struct __PyObjectEncoder {
 enum PANDAS_FORMAT { SPLIT, RECORDS, INDEX, COLUMNS, VALUES };
 
 int PdBlock_iterNext(JSOBJ, JSONTypeContext *);
+static Py_ssize_t get_attr_length(PyObject *obj, char *attr);
```

Comment:
Rather than forward declaring this can you just swap the position the of `get_attr_length` and `is_simple_frame` functions?

Reply:
Will make that change